jib-core/src/test/java/com/google/cloud/tools/jib/http/TestWebServer.java

```diff
@@ -59,10 +68,10 @@ private void waitUntilReady() throws IOException, InterruptedException {
   private void serve200() {
     try {
       while (true) {
-        try (Socket socket = serverSocket.accept();
-            OutputStream out = socket.getOutputStream()) {
-          out.write("HTTP/1.1 200 OK\n\nHello World!".getBytes(StandardCharsets.UTF_8));
-        }
+        Socket socket = serverSocket.accept();
+        sockets.add(socket);
+        OutputStream out = socket.getOutputStream();
```

Comment:
I think the issue is that the `OutputStream` needs to be closed first.

Reply:
I don't really think so... closing socket should just close anything underneath it gracefully. Anyway, the test is now passing, so I think my theory about keep-alive seems right.

Reply:
But you could be also right. It'd be interesting to test closing `OutputStream` first on Windows.

Reply:
Actually, nevermind, the try-with-resource closes resources with the reverse order of creation.